task_15/app.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887245)
```suggestion
const express = require("express")
```